acoustid/data/submission.py

```
@@ -1,12 +1,14 @@
 # Copyright (C) 2011 Lukas Lalinsky
 # Distributed under the MIT license, see the LICENSE file for details.
 
+import array
 import datetime
 import logging
 import uuid
```

Ruff (0.8.2)

`uuid` imported but unused

Remove unused import: `uuid`

(F401)